src/data/actions/assets.js

```diff
@@ -20,6 +22,12 @@ export const filterUpdate = (filterKey, filterValue) => ({
   data: { [filterKey]: filterValue },
 });
 
+export const sortUpdate = (sort, direction) => ({
```

Comment:
sortUpdate is an action creator that creates an action of type SORT UPDATE and with data that is an object containing the sort and direction.